#include<exception>

class ExceptionError : public std::exception
{

};